apps/web/modules/availability/[schedule]/schedule-view.tsx

```diff
@@ -114,20 +102,12 @@ export const AvailabilitySettingsWebWrapper = ({
     },
   });
 
-  // TODO: reimplement Skeletons for this page in here
-  if (isPending) return null;
-
-  // We wait for the schedule to be loaded before rendering the form inside AvailabilitySettings
-  // since `defaultValues` cannot be redeclared after first render and using `values` will
-  // trigger a form reset when revalidating. Introducing flaky behavior.
-  if (!schedule) return null;
```

Comment:
schedule will always be there. Otherwise, `not-found` page would have been already called